internal/api/sms_provider/twilio.go

```diff
@@ -70,7 +80,9 @@ func (t *TwilioProvider) SendSms(phone, message, channel string) (string, error)
 	receiver := "+" + phone
 	if channel == WhatsappProvider {
 		receiver = channel + ":" + receiver
-		sender = channel + ":" + sender
+		if !isTwilioMessagingServiceID(sender) {
```

Comment:
maybe we can just add a comment here to indicate that the `MessageServiceSid` can be a phone number in which case, we need to provide a different format for the sender by prefixing it with the channel?

Reply:
I think we can change the regexp to check for phone numbers which might be clearer.  Under that scenario if it's a Messaging Service it'll go through and if not Twilio will return an error.

We only append `whatsapp:` if the MessagingServiceID is a phone number.